app/sagas/session.js

```diff
@@ -13,14 +15,48 @@ export function* autoJoinUser(action) {
             sessionId,
             user: currentUser
         }});
+        storeSessionToLocalStorage(currentUser, sessionId);
     }
 }
 
+export function* renameCurrentSessionInLocalStorage (action) {
+    console.log("renaming session");
+    console.log(action);
+    const currentSession = yield select(state => state.session.id);
+    const currentUser = yield select(state => state.user.name);
+    let savedSessions = ls.get('sessions');
+    let savedSession  = find(savedSessions[currentUser], session => session.id === currentSession);
+    savedSession.name = action.payload;
+    ls.set('sessions', savedSessions);
+}
+
+function storeSessionToLocalStorage(currentUser, sessionId) {
+    let savedSessions = ls.get('sessions');
```

Comment:
Do we want to store these sessions against a user? A browser is (most of the time) related to one given user, so we could possibly simplify this and just store the sessions, not the user, and always return the full list of sessions?